example/src/main/tut/data/Free.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: "Free"
+---
+
+# Free
+
+Free is a recursive data structure that either returns a vale `A` or returns an effect computation in `F[_]` that yields
```

Comment:
"value", "effectful".

That said, I think this makes `Free` sound like it's limited to representing computations. It's also just an "`F`-branching tree", which this doesn't make clear.